### PR TITLE
Create FileIo class

### PR DESCRIPTION
**Pull request by NoahZinter**
### Changes

- Description of changes
Create/test FileIO class 

### Requested review on...
- Specifics on, logic, naming schemes, etc.

### Checklist
- [ x] Tests pass

closes # 58

### CODE REVIEW

File: spec/file_io_spec.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  it 'returns an array' do 
+    mock_row = {
+      id: 12_345,
+      name: 'Widget',
+      description: 'Item desc',
+      unit_price: '12.23',
+      merchant_id: '12345'
+    }
```
Comment (sami-p) [inline spec/file_io_spec.rb:21]: So this will return an array of the mock_row hash?

**Comment by mkrumholz** [comment]
We should run rubocop on this -- looks like we've got some spacing stuff going on.

**Review by sami-p — APPROVED** [review]

File: lib/file_io.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class FileIo
+  def self.process_csv(filename, type)
+    elements = []
```
Comment (mkrumholz) [inline lib/file_io.rb:3]: I wonder if there will be a refactor warranted here to not require the accumulator. We should leave that to a later time, but if y'all know of something we could use, let's discuss!

File: spec/file_io_spec.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    allow(CSV).to receive(:foreach).and_yield(mock_row)
```
Comment (mkrumholz) [inline spec/file_io_spec.rb:23]: I think the spacing here could use a refresh. Let's get together as a group on conventions and fix this for all test files.

**Review by mkrumholz — APPROVED** [review]
Excited to use this to refactor our SalesEngine and Repo classes

**Pull request merged**